chromeipass/options/options.html

```diff
@@ -173,6 +173,12 @@ <h3 class="muted"><img src="/icons/keepass_48x48.png" alt="logo" /> chromeIPass<
 				<hr />
 				<p>
 					<label class="checkbox">
+						<input type="checkbox" name="autoRetrieveCredentials" value="1" /> Automatically retrieve credentials.
```

Comment:
There needs to be a corresponding change in `background/page.js` such that it initializes `autoRetrieveCredentials` to 1 if it is not already set. Please update, and the PR looks good.